--- Running scipy.fft.idst#9YY/idstYY, strategy: relancer
[ERR MSG]: OrderedDict([('project', 'scipy.fft.idst#9YY'), ('notebook', 'idstYY'), ('text', '')])
[INFO] This case is fully fixed!


real	0m0.452s
user	0m1.959s
sys	0m6.881s
